modelcontextprotocol/server.py

```diff
@@ -358,9 +371,73 @@ def update_assets_tool(
                     "qualified_name": "Asset Qualified Name 2"
                 }
             ],
-            attribute_name="user_description",
+            attribute_name="readme",
+            attribute_values=[
+                "<h1>New readme content for asset 1</h1>",
+                "<h1>New readme content for asset 2</h1>"
+            ]
+        )
+
+        # Update readme for a single asset with Markdown
+        result = update_assets_tool(
+            assets={
+                "guid": "asset-guid-here",
+                "name": "Asset Name",
+                "type_name": "Asset Type Name",
+                "qualified_name": "Asset Qualified Name"
+            },
+            attribute_name="readme",
+            attribute_values=['# Data Pipeline Overview\n\n'
+                            '## Purpose\n'
+                            'This pipeline processes customer transaction data for analytics.\n\n'
+                            '### Key Features\n'
+                            '- **Real-time Processing**: Handles streaming data\n'
+                            '- **Data Quality**: Implements [validation rules](https://docs.example.com/validation)\n'
```

Comment:
Lets limit to a single example and keep the example short to not cause context issues